documentation/dev_running_from_source.markdown

```diff
@@ -159,6 +159,24 @@ run the following:
 
     $ lein run -c /path/to/puppetserver.conf
 
+Running the Agent
+-----
+
+Use a command like the one below to run an agent against your running puppetserver:
+
+        puppet agent --confdir ~/.puppetlabs/etc/puppet \
+                     --debug -t
+
+Note that a system installed Puppet Agent is ok for use with
+source-based PuppetDB and Puppet Server. The `--confdir` above
+specifies the same confdir that Puppet Server is using. Since the
+Puppet Agent and Puppet Server instances are both using the same
+confdir, they're both using the same certificates as well. This
+alleviates the need to sign certificates as a separate step. The
+`--server` flag will also indicate the host puppetserver is on. If you
```

Comment:
@senior Ping.  You have time to remove this comment?  I can merge it after that.